app/pages/sections/catalogue-solutions/routes.js

```diff
@@ -20,5 +22,16 @@ export const catalogueSolutionsRoutes = (authProvider, addContext) => {
     return res.render('pages/sections/catalogue-solutions/catalogue-solutions/template.njk', addContext({ context, user: req.user, csrfToken: req.csrfToken() }));
   }));
 
+  router.post('/', authProvider.authorise({ claim: 'ordering' }), withCatch(authProvider, async (req, res) => {
+    const { orderId } = req.params;
+
+    await putCatalogueSolutions({
+      orderId,
+      accessToken: extractAccessToken({ req, tokenType: 'access' }),
+    });
+
+    return res.redirect(`${config.baseUrl}/organisation/${orderId}`);
```

Comment:
I guess this is going to change later?

Reply:
No, this will always be this I believe. Clicking on the Add Catalogue Solution will do something else. But clicking the Continue button will always redirect back to the task-list page. Bit of a strange flow on this section.

Reply:
where does the put to ordapi with service recipients occur?